wf/ja-jp/content/topics/workflows/connector-builder/about-functions.htm

```diff
@@ -122,7 +122,7 @@
                                 </div>
                             </div>
                             <div class="title-bar-layout outer-row">
-                                <div class="logo-wrapper"><a class="logo" href="https://support.okta.com/help/s/?language=ja" alt="Okta"></a>
+                                <div class="logo-wrapper"><a class="logo" href="https://help.okta.com/okta_help.htm?id=csh-index" alt="Okta"></a>
```

Comment:
same question about the redirect as in the other PRs, will it redirect to Japanese content?